### DISCUSSION TITLE
Web UI hangs after switching to control tab

### DISCUSSION THREAD

**Discussion by melston** [feelfreelinux/octo4a#318] [Q&A]
I'm running octo4a on my old Samsung Tab A and I have successfully printed once with it.  But, when using the Web UI on my laptop, I cannot switch to the control tab as the entire page locks up.  All I can do is delete the tab for that page and reload the UI in a new tab.  I don't have any such problem with the other tabs on the UI.  I suspect it has something to do with the camera view as that doesn't always load and it doesn't update when it does load.

Also, is there a fix for the Ender 3 V2 temperature parsing issue for octo4a?  The only thing I found is a python script but I don't see how to use that with octo4a.

**Comment by Smarti0101** [comment]
Same problem on my Galaxy S6Edge.
